protocols/tck/cloudstate/tck/model/eventlogeventing.proto

```diff
@@ -0,0 +1,178 @@
+import "cloudstate/eventing.proto";
+import "cloudstate/entity_key.proto";
+
+//
+// The `EventLogSubscriberModel` service is an Action that should be implemented in the following ways:
+//
+// - The `ProcessEventOne` method receives an `EventOne` message, and must behave according to the passed in step.
+// - The `ProcessEventTwo` method receives an `EventTwo` message, and must return a stream of responses, one for each
+//   incoming step.
+// - The `Effect` method receives an `EffectRequest` message and must respond with a `Response` that contains the id
+//   from the effect message.
+// - The `ProcessAnyEvent` method receives a `google.protobuf.Any`, which will contain JSON serialized according to
+//   the Cloudstate JSON serialization conventions, with a `type_url` of `json.cloudstate.io/JsonEvent`. The contents
+//   of the JSON message will be a JSON object with a single message property, and the call must respond with this
+//   message in the `Response`.
+// - Forwarding and side effects should be made to the `Effect` call.
+// - The response to the Process calls, or the effects and forwards emitted by them, must contain the message field
+//   from the incoming events, along with the id read from the CloudEvent metadata source property.
+//
+service EventLogSubscriberModel {
+  rpc ProcessEventOne(EventOne) returns (Response) {
+    option (.cloudstate.eventing) = {
+              in: {
+                  event_log: "eventlogeventing-one"
+              }
+    };
+  };
+
+  rpc ProcessEventTwo(EventTwo) returns (stream Response) {
+    option (.cloudstate.eventing) = {
+              in: {
+                  event_log: "eventlogeventing-one"
+              }
+    };
+  };
+
+  rpc Effect(EffectRequest) returns (Response);
+
+  rpc ProcessAnyEvent(google.protobuf.Any) returns (Response) {
+    option (.cloudstate.eventing) = {
+              in: {
+                  event_log: "eventlogeventing-two"
+              }
+    };
+  };
+}
+
+
+//
+// The `EventSourcedEntityOne` service is an event sourced entity that should be implemented in the following ways:
+//
+// - The `EmitEvent` method should emit the event in the `EmitEventRequest` method as a protobuf serialized event.
+// - The persistence id for it must be `eventlogeventing-one`.
+//
+service EventSourcedEntityOne {
+  rpc EmitEvent(EmitEventRequest) returns (google.protobuf.Empty);
+}
+
+//
+// The `EventSourcedEntityTwo` service is an event sourced entity that should be implemented in the following ways:
+//
+// - The `EmitJsonMethod` method should emit an event serialised as JSON. This event should:
+//   - Contain a single `message` property with the value of the `message` field in `JsonEvent`.
+//   - Be serialized according to the Cloudstate JSON serialization conventions - that is, with the JSON serialized to
+//     bytes, then placed into a protobuf message with a single bytes field with field number 1.
+//   - Have a type_url of `json.cloudstate.io/JsonEvent`.
```

Comment:
The Java Support implementation does return here a `JsonMessage`, instead of a `JsonEvent`as an Event.
Could we write that the type_url has to have a prefix of ``json.cloudstate.io/`?